src/controllers/Events/deleteEvent.js

```diff
@@ -1,5 +1,18 @@
+
+/**
+ * @module Delete Event Controller
+ * @param {Request} req - HTTP Request from the client
+ * @param {Response} res - HTTP Response for the client
+ * 
+ * @description
+ * This controller will allow the user to delete an event of his own, if all parameters are correct.
+ * 
+ * @todo
+ * Nothing for now.
+ */
+
+
 module.exports.deleteEvent = (req, res) => {
-  if (!req.user) return res.status(403).send("Not Logged In.");
```

Comment:
We do not need to check if a user is logged in in the controller anymore, the reason is that we've built a middelware called `verifyLogin` to do that for any controller we added it to.

Hope that is clear.
Massiles.